generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics.component.ts.ejs

```diff
@@ -17,46 +17,47 @@
  limitations under the License.
 -%>
 import {Component, OnInit} from '@angular/core';
+import { flatMap } from 'rxjs/operators';
 
-import { <%=jhiPrefixCapitalized%>MetricsService } from './metrics.service';
+import { MetricsService, Metrics, MetricsKey, ThreadDump, Thread } from './metrics.service';
 
 @Component({
```

Comment:
Can't we use the OnPush here ?
I see that you used functions inside `ngIf` so it would be much better in terms of performance to use the OnPush strategy here

Reply:
Switched to OnPush now